davidagitlen/Tripedia-fe
# co-authored-by: Chris Lane <c.lane.dev@gmail.com>

co-authored-by: Chris Lane <c.lane.dev@gmail.com>
co-authored-by: Glynnis O'Connell <44204866+GlynnisOC@users.noreply.github.com>
co-authored-by: Brandy Mello <brandyamello@gmail.com>
co-authored-by: Paul Schlattmann <3031146+pschlatt@users.noreply.github.com>

#### What’s this PR do?

Updates all functionality to dynamically generate categories for each form and allow user to toggle points off and on based on the categories they want to see. 

#### Where should the reviewer start?

Changes have taken place in the util directory, as well as in each form, the map and pin components, and apiCalls. 

#### How should this be manually tested?

Either making a fetch from the startForm with a user-entered origin and destination, or pulling in the data from the jsonData file should allow you to see a route. 

#### Any background context you want to provide?

#### What are the relevant tickets?

#29 

#### Screenshots (if appropriate)

#### Questions:

# Implements/Fixes:
* What issue does this close?
